pkg/specs/apptype/determine_type.go

```diff
@@ -80,16 +81,35 @@ func (r *inspector) DetermineApplicationType(ctx context.Context, upstream strin
 		return &localAppCopy{AppType: "runbook.replicated.app", LocalPath: parts[0]}, nil
 	}
 
-	r.ui.Info(fmt.Sprintf("Attempting to retrieve upstream %s ...", upstream))
+	if i.viper.GetBool("isEdit") {
```

Comment:
I think in general we're trying to avoid new Viper.GetXXX calls at runtime -- is there a reason this can't be set as a member field on `inspector` during construction?